src/coreComponents/physicsSolvers/multiphysics/CoupledReservoirAndWellsBase.hpp

```diff
@@ -187,6 +191,10 @@ class CoupledReservoirAndWellsBase : public CoupledSolver< RESERVOIR_SOLVER, WEL
 
     DomainPartition & domain = this->template getGroupByPath< DomainPartition >( "/Problem/domain" );
 
+    // Validate well perforations: Ensure that each perforation is in a region targetted by the solver
+    if( !validateWellPerforations( domain ))
+      return;
```

Comment:
```suggestion
    if( !validateWellPerforations( domain ))
    {
      return;
    }
```
This is a nitpicking comment sorry, but this is the convention that we use in GEOS (at least, that is my understanding)

Reply:
I had assumed this would be handled by uncrustify.